spec/controllers/contacts_controller_spec.rb

```diff
@@ -34,12 +34,5 @@
       @user_instance = class_double('User').as_stubbed_const
       allow(@user_instance).to receive(:all).and_return([@user_instance, @user_instance, @user_instance])
     end
-
-    it 'GET #needs' do
```

Comment:
Hm, was this removed because it's no longer relevant? Or do we need to try and get it to pass again?

Reply:
i think there was no longer a seperate test view at the time i deleted that